lib/Mojo/DOM.pm

```diff
@@ -686,6 +686,9 @@ Trailing key/value pairs can be used to declare xml namespace aliases.
 
 Find this element's namespace, or return C<undef> if none could be found.
 
+  # 'http://www.w3.org/2000/svg'
```

Comment:
I believe other examples use double quotes.

Reply:
Does that example even work? I had to do 

```
Mojo::DOM->new('<svg xmlns:svg="http://www.w3.org/2000/svg"><svg:circle>3.14</svg:circle></svg>')->at('circle')->namespace;
```

to get the result you see.